src/automation/azext_automation/manual/custom.py

```diff
@@ -135,3 +135,203 @@ def automation_job_stop(client, resource_group_name, automation_account_name, jo
+        if included_kb_numbers is not None:
+            linux['includedKbNumbers'] = included_kb_numbers
+        if reboot_setting is not None:
+            linux['rebootSetting'] = reboot_setting
+        update_configuration['linux'] = linux
+    if duration is not None:
+        update_configuration['duration'] = duration
+    if azure_virtual_machines is not None:
+        update_configuration['azureVirtualMachines'] = azure_virtual_machines
+    if non_azure_computer_names:
+        update_configuration['nonAzureComputerNames'] = non_azure_computer_names
+
+    targets = {}
+    azure_queries = {}
+    if azure_queries_scope is not None:
+        azure_queries['scope'] = azure_queries_scope
+    if azure_queries_locations is not None:
+        azure_queries['locations'] = azure_queries_locations
+    if tags is not None:
+        azure_queries['tagSettings'] = {'tags': {'tag': tags}}
+    if azure_queries:
+        targets['azureQueries'] = [azure_queries]
+
+    non_azure_queries = {}
+    if non_azure_queries_function_alias is not None:
+        non_azure_queries['functionAlias'] = non_azure_queries_function_alias
+    if non_azure_queries_workspace_id is not None:
+        non_azure_queries['workspaceId'] = non_azure_queries_workspace_id
+    if non_azure_queries:
+        targets['nonAzureQueries'] = [non_azure_queries]
+
+    if targets:
+        update_configuration['targets'] = targets
+    if update_configuration:
+        parameters['updateConfiguration'] = update_configuration
+
+    schedule_info = {}
+    if start_time is not None:
+        schedule_info['startTime'] = start_time
+    if expiry_time is not None:
+        schedule_info['expiryTime'] = expiry_time
+    if expiry_time_offset_minutes is not None:
+        schedule_info['expiryTimeOffsetMinutes'] = expiry_time_offset_minutes
+    if is_enabled is not None:
+        schedule_info['isEnabled'] = is_enabled
+    if next_run is not None:
+        schedule_info['nextRun'] = next_run
+    if next_run_offset_minutes is not None:
+        schedule_info['nextRunOffsetMinutes'] = next_run_offset_minutes
+    if interval is not None:
+        schedule_info['interval'] = interval
+    if frequency is not None:
+        schedule_info['frequency'] = frequency
+    if time_zone is not None:
+        schedule_info['time_zone'] = time_zone
+    if creation_time is not None:
+        schedule_info['creationTime'] = creation_time
+    if last_modified_time is not None:
+        schedule_info['lastModifiedTime'] = last_modified_time
+    if description is not None:
+        schedule_info['description'] = description
+    if schedule_info:
+        parameters['scheduleInfo'] = schedule_info
+
+    tasks = {}
+    pre_task = {}
+    if pre_task_status is not None:
+        pre_task['status'] = pre_task_status
+    if pre_task_source is not None:
+        pre_task['source'] = pre_task_source
+    if pre_task_status is not None:
+        pre_task['job_id'] = pre_task_job_id
```

Comment:
The attribute name is usually hump style. Can you confirm that the attribute name is `job_id` instead of `jobId`?